src/platform/qt/SettingsView.ui

```diff
@@ -780,6 +780,16 @@
          </property>
         </widget>
        </item>
+       <item row="4" column="1">
+        <widget class="QCheckBox" name="showFilenameInLibrary">
+         <property name="toolTip">
+          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;If the rom does not match mGBA's internal database, filename will be shown regardless if this option is unchecked.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
+         </property>
```

Comment:
mGBA doesn't have tooltips for stuff yet, so let's keep that consistent for now at least.

Reply:
I saw it didn't had but I added it because I felt necessary mainly because this particular option needs some clarification. And also to start a discussion on how to give more feedback to the user.

The behavior isn't clear to the end-user. When he sees "Name" in the library, convention says he expects this:
![image](https://user-images.githubusercontent.com/1557903/160277374-fd3f79a2-27dd-4711-8194-509c882b8f86.png)
So, adding an option that explicitly changes this behavior, yet does not change under certain circunstances (like not having the database) and not making it clear to the user will lead to confusion. That is why I felt necessary to add a tooltip to make it clear that, if the rom cannot be found in the database, it will default to the filename regardless of the checkbox.